package.json

```diff
@@ -24,8 +24,9 @@
 		"clean-extensions": "rm -rf _inc/blocks/ ",
 		"install-if-deps-outdated": "yarn check 2> /dev/null || yarn install --check-files --production=false --frozen-lockfile",
 		"distclean": "rm -rf node_modules && yarn clean",
-		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-client && yarn build-php && yarn build-extensions && yarn build-search",
+		"build": "yarn install-if-deps-outdated && yarn clean && yarn build-client && yarn build-php && yarn build-extensions && yarn build-search && yarn build-packages",
 		"build-client": "gulp",
+		"build-packages": "webpack --config ./webpack.config.packages.js",
```

Comment:
We may have to add this to the `build-production` command as well.

Reply:
I completely missed that one. I have now added. Thank you